WormsApp/makefile

```diff
@@ -1,2 +1,2 @@
 all:
-	g++ -std=c++14 -o wormApp.e main.cpp graphics.cpp worm.cpp board.cpp `pkg-config gtkmm-3.0 --cflags --libs` -Wall -Wextra -Werror
+	g++ -std=c++14 -o wormApp.e main.cpp graphics.cpp worm.cpp board.cpp `pkg-config gtkmm-3.0 --cflags --libs` #-Wall -Wextra -Werror

```

Comment:
These flags are useful. They usually point some minor issues, but sometimes it may catch something you wouldn't notice.

Reply:
Uncommented